### DISCUSSION TITLE
Masanın üzerindeki paralar sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#89] [Zeka Sorusu]
Bir masa üzerinde 100 adet 1 TL var. Bunlardan 80’i yazı 20’si tura. Gözleriniz bağlı ve dokunarak paranın yazı mı, tura mı olduğunu anlayamıyorsunuz. Sizden istenen şu;
Bu yüz adet parayı öyle bir yöntemle iki gruba ayırınız. Her iki grupta da tura sayısı eşit olsun ve siz bundan emin olun.

**Comment by sonsuzus** [comment]
Bu sorudan itibaren.

**Comment by tasgin123** [comment]
iİlk önce, 100 adet parayı rastgele 20 lik ve 80 lik iki gruba ayırıyoruz.
oluşabilecek rastgele bir ihtimali denersek

1. ihtimalde ilk grupta 18 tura ve 2 yazı, ikinci grupta da 2 tura ve 78 yazı olsun.
sonra da ilk gurubu tamamen ters çeviriyoruz.
1.gurup 18 yazı 2 tura, 2. gurup da  2 tura ve 78 yazı oluyor ve böylece eşitleniyor

her ihtimalde de e-bu aynen böyle oluyor
